components/proxy/plugins/configcat/configcat.go

```diff
@@ -64,19 +68,44 @@ func (ConfigCat) CaddyModule() caddy.ModuleInfo {
 	}
 }
 
+func ServeFromFile(w http.ResponseWriter, r *http.Request, fileName string) {
+	fp := path.Join(configCatConfigDir, fileName)
+	d, err := os.Stat(fp)
```

Comment:
This will result in a file system call on every request. Depending how heavily this gets hit, it can cause unnecessary interrupts for the application. If it becomes a problem, an optimization is to re-read this file every minute and serve from a local in-memory version.